var/spack/repos/builtin/packages/nodejs/package.py

```diff
@@ -0,0 +1,44 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Nodejs(AutotoolsPackage):
+    """Node.js JavaScript runtime"""
+
+    homepage = "https://nodejs.org/"
+    url      = "https://nodejs.org/dist/v4.5.0/node-v4.5.0.tar.gz"
+
+    version('4.5.0', '1885586b4b8a2263f77dce27855661e9')
+
+    depends_on('python')
+    depends_on('openssl')
+
+    def install(self, spec, prefix):
```

Comment:
This must go, see documentation/other packages on which functions to override. I am AFK, can't copy-paste